std/experimental/allocator/package.d

```diff
@@ -2495,20 +2499,20 @@ class CSharedAllocatorImpl(Allocator, Flag!"indirect" indirect = No.indirect)
     processAllocator.dispose(p);
 
     // Create an array of 50 doubles initialized to -1.0
-    double[] arr = theAllocator.makeArray!double(50, -1.0);
+    double[] arr = threadAllocator.makeArray!double(50, -1.0);
 
     // Check internal pointer
     void[] result;
-    assert(theAllocator.resolveInternalPointer(null, result) == Ternary.no);
-    Ternary r = theAllocator.resolveInternalPointer(arr.ptr, result);
+    assert(threadAllocator.resolveInternalPointer(null, result) == Ternary.no);
+    Ternary r = threadAllocator.resolveInternalPointer(arr.ptr, result);
     assert(result.ptr is arr.ptr && result.length >= arr.length);
 
     // Append two zeros to it
-    theAllocator.expandArray(arr, 2, 0.0);
+    threadAllocator.expandArray(arr, 2, 0.0);
     // On second thought, take that back
-    theAllocator.shrinkArray(arr, 2);
+    threadAllocator.shrinkArray(arr, 2);
     // Destroy and deallocate
-    theAllocator.dispose(arr);
+    threadAllocator.dispose(arr);
 }
 
 __EOF__
```

Comment:
Just noticed this. I know we shouldn't mix up PR topics, but any ideas on what this is doing here?